<h1 id="references" class="unnumbered">References</h1>
<div id="refs" class="references">
<div id="ref-polonsky2019outbreak">
<p>Polonsky, J. A., Baidjoe, A., Kamvar, Z. N., Cori, A., Durski, K., Edmunds, W. J., Eggo, R. M., Funk, S., Kaiser, L., Keating, P., Polain de Waroux, O. le, Marks, M., Moraga, P., Morgan, O., Nouvellet, P., Ratnayake, R., Roberts, C. H., Whitworth, J., &amp; Jombart, T. (2019). Outbreak analytics: A developing data science for informing the response to emerging pathogens. <em>Philosophical Transactions of the Royal Society B: Biological Sciences</em>, <em>374</em>(1776), 20180276. <a href="https://doi.org/10.1098/rstb.2018.0276">https://doi.org/10.1098/rstb.2018.0276</a></p>
</div>
<div id="ref-miorini2019differential">
<p>Miorini, T. J. J., Kamvar, Z. N., Higgins, R. S., Raetano, C. G., Steadman, J. R., &amp; Everhart, S. E. (2019). Differential aggressiveness of <em>Sclerotinia sclerotiorum</em> isolates from North and South America and partial host resistance in Brazilian soybean and dry bean cultivars. <em>Tropical Plant Pathology</em>, <em>44</em>(1), 73–81. <a href="https://doi.org/10.1007/s40858-018-00273-w">https://doi.org/10.1007/s40858-018-00273-w</a></p>
</div>
<div id="ref-kamvar2019epidemic">
<p>Kamvar, Z. N., Cai, J., Pulliam, J. R., Schumacher, J., &amp; Jombart, T. (2019). Epidemic curves made easy using the R package <em>incidence</em> [version 1; peer review: 1 approved, 2 approved with reservations]. <em>F1000Research</em>, <em>8</em>, 139. <a href="https://doi.org/10.12688/f1000research.18002.1">https://doi.org/10.12688/f1000research.18002.1</a></p>
</div>
<div id="ref-pannullo2018genetic">
<p>Pannullo, A., Kamvar, Z. N., Miorini, T. J. J., Steadman, J. R., &amp; Everhart, S. E. (2018). Genetic variation and structure of <em>Sclerotinia sclerotiorum</em> populations from soybean in Brazil. <em>Tropical Plant Pathology</em>, <em>44</em>(1), 53–64. <a href="https://doi.org/10.1007/s40858-018-0266-5">https://doi.org/10.1007/s40858-018-0266-5</a></p>
</div>
<div id="ref-kamvar2018something">
<p>Kamvar, Z. N., &amp; Everhart, S. E. (2018). Something in the agar does not compute: On the discriminatory power of mycelial compatibility in <em>Sclerotinia sclerotiorum</em>. <em>Tropical Plant Pathology</em>, <em>44</em>(1), 32–40. <a href="https://doi.org/10.1007/s40858-018-0263-8">https://doi.org/10.1007/s40858-018-0263-8</a></p>
</div>
<div id="ref-moraga2018epiflows">
<p>Moraga, P., Dorigatti, I., Kamvar, Z. N., Piatkowski, P., Toikkanen, S. E., Nagraj, V., Donnelly, C. A., &amp; Jombart, T. (2018). <em>epiflows</em>: An R package for risk assessment of travel-related spread of disease [version 1; peer review: 2 approved with reservations]. <em>F1000Research</em>, <em>7</em>, 1374. <a href="https://doi.org/10.12688/f1000research.16032.1">https://doi.org/10.12688/f1000research.16032.1</a></p>
</div>
<div id="ref-kamvar2017population">
<p>Kamvar, Z. N., Amaradasa, B. S., Jhala, R., McCoy, S., Steadman, J. R., &amp; Everhart, S. E. (2017). Population structure and phenotypic variation of <em>Sclerotinia sclerotiorum</em> from dry bean (<em>Phaseolus vulgaris</em>) in the United States. <em>PeerJ</em>, <em>5</em>, e4152. <a href="https://doi.org/10.7717/peerj.4152">https://doi.org/10.7717/peerj.4152</a></p>
</div>
<div id="ref-grunwald2017best">
<p>Grünwald, N. J., Everhart, S., Knaus, B. J., &amp; Kamvar, Z. N. (2017). Best practices for population genetic analyses. <em>Phytopathology</em>, <em>107</em>(9), 1000–1010. <a href="https://doi.org/10.1094/PHYTO-12-16-0425-RVW">https://doi.org/10.1094/PHYTO-12-16-0425-RVW</a></p>
</div>
<div id="ref-jombart2017apex">
<p>Jombart, T., Archer, F., Schliep, K., Kamvar, Z., Harris, R., Paradis, E., Goudet, J., &amp; Lapp, H. (2017). apex: Phylogenetics with multiple genes. <em>Molecular Ecology Resources</em>, <em>17</em>(1), 19–26. <a href="https://doi.org/10.1111/1755-0998.12567">https://doi.org/10.1111/1755-0998.12567</a></p>
</div>
<div id="ref-kamvar2017developing">
<p>Kamvar, Z. N., López-Uribe, M. M., Coughlan, S., Grünwald, N. J., Lapp, H., &amp; Manel, S. (2017). Developing educational resources for population genetics in R: An open and collaborative approach. <em>Molecular Ecology Resources</em>, <em>17</em>(1), 120–128. <a href="https://doi.org/10.1111/1755-0998.12558">https://doi.org/10.1111/1755-0998.12558</a></p>
</div>
<div id="ref-kamvar2016development">
<p>Kamvar, Z. N. (2016, December). <em>Development and application of tools for genetic analysis of clonal populations</em> (PhD thesis). Retrieved from <a href="https://ir.library.oregonstate.edu/concern/graduate_thesis_or_dissertations/dz010t67t">https://ir.library.oregonstate.edu/concern/graduate_thesis_or_dissertations/dz010t67t</a></p>
</div>
<div id="ref-grunwald2016first">
<p>Grünwald, N., Larsen, M., Kamvar, Z., Reeser, P., Kanaskie, A., Laine, J., &amp; Wiese, R. (2016). First report of the EU1 clonal lineage of <em>Phytophthora ramorum</em> on tanoak in an Oregon forest. <em>Plant Disease</em>, <em>100</em>(5), 1024–1024. <a href="https://doi.org/10.1094/PDIS-10-15-1169-PDN">https://doi.org/10.1094/PDIS-10-15-1169-PDN</a></p>
</div>
<div id="ref-tabima2016microbe">
<p>Tabima, J. F., Everhart, S. E., Larsen, M. M., Weisberg, A. J., Kamvar, Z. N., Tancos, M. A., Smart, C. D., Chang, J. H., &amp; Grünwald, N. J. (2016). Microbe-ID: An open source toolbox for microbial genotyping and species identification. <em>PeerJ</em>, <em>4</em>, e2279. <a href="https://doi.org/10.7717/peerj.2279">https://doi.org/10.7717/peerj.2279</a></p>
</div>
<div id="ref-grunwald2015population">
<p>Grünwald, N. J., Kamvar, Z. N., &amp; Everhart, S. E. (2015). Population genetics in r. Retrieved from <a href="https://grunwaldlab.github.io/Population_Genetics_in_R/">https://grunwaldlab.github.io/Population_Genetics_in_R/</a></p>
</div>
<div id="ref-kamvar2015spatial">
<p>Kamvar, Z., Larsen, M., Kanaskie, A., Hansen, E., &amp; Grünwald, N. (2015). Spatial and temporal analysis of populations of the sudden oak death pathogen in Oregon forests. <em>Phytopathology</em>, <em>105</em>(7), 982–989. <a href="https://doi.org/10.1094/PHYTO-12-14-0350-FI">https://doi.org/10.1094/PHYTO-12-14-0350-FI</a></p>
</div>
<div id="ref-kamvar2015novel">
<p>Kamvar, Z. N., Brooks, J. C., &amp; Grünwald, N. J. (2015). Novel R tools for analysis of genome-wide population genetic data with emphasis on clonality. <em>Frontiers in Genetics</em>, <em>6</em>, 208. <a href="https://doi.org/10.3389/fgene.2015.00208">https://doi.org/10.3389/fgene.2015.00208</a></p>
</div>
<div id="ref-weiland2015population">
<p>Weiland, J. E., Garrido, P., Kamvar, Z. N., Espı'ndola, A. S., Marek, S. M., Grünwald, N. J., &amp; Garzón, C. D. (2015). Population structure of <em>Pythium irregulare</em>, <em>P.sylvaticum</em>, and <em>P. ultimum</em> in forest nursery soils of Oregon and Washington. <em>Phytopathology</em>, <em>105</em>(5), 684–694. <a href="https://doi.org/10.1094/PHYTO-05-14-0147-R">https://doi.org/10.1094/PHYTO-05-14-0147-R</a></p>
</div>
<div id="ref-kamvar2014poppr">
<p>Kamvar, Z. N., Tabima, J. F., &amp; Grünwald, N. J. (2014). Poppr: An R package for genetic analysis of populations with clonal, partially clonal, and/or sexual reproduction. <em>PeerJ</em>, <em>2</em>, e281. <a href="https://doi.org/10.7717/peerj.281">https://doi.org/10.7717/peerj.281</a></p>
</div>
<div id="ref-buckner2007involving">
<p>Buckner, B., Beck, J., Browning, K., Fritz, A., Grantham, L., Hoxha, E., Kamvar, Z., Lough, A., Nikolova, O., Schnable, P. S., &amp; others. (2007). Involving undergraduates in the annotation and analysis of global gene expression studies: Creation of a maize shoot apical meristem expression database. <em>Genetics</em>, <em>176</em>(2), 741–747. <a href="https://doi.org/10.1534/genetics.106.066472">https://doi.org/10.1534/genetics.106.066472</a></p>
</div>
</div>
